package.json

```diff
@@ -60,6 +61,7 @@
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "eslint .",
+    "eslint-fix": "eslint . --fix",
```

Comment:
Why not just type `npx eslint . --fix` instead of `npm run eslint-fix`? It's even less keystrokes.